packages/beta/src/types.ts

```diff
@@ -322,3 +330,72 @@ export interface Subscription {
   subscriberExternalId?: CogniteExternalId;
   metadata?: Metadata;
 }
+
+export interface DatapointsMultiQuery
+  extends Omit<DataPointsMultiQueryStable, 'items'> {
+  items: DatapointsQuery[];
+}
+
+export type DatapointsQuery = DatapointsQueryId | DatapointsQueryExternalId;
+
+export interface DatapointsQueryExternalId
+  extends DatapointsQueryProperties,
+    ExternalId {}
+
+export interface DatapointsQueryId
+  extends DatapointsQueryProperties,
+    InternalId {}
+
+export type Aggregate = AggregateStable | 'countUncertain' | 'countBad';
+export interface DatapointsQueryProperties
+  extends Omit<DatapointsQueryPropertiesStable, 'aggregates'> {
+  /**
+   * The aggregates to be returned. This value overrides top-level default aggregates list when set. Specify all aggregates to be retrieved here. Specify empty array if this sub-query needs to return datapoints without aggregation.
+   */
+  aggregates?: Aggregate[];
+  /**
+   * denotes that the status of the data point should be included in the response.
+   */
+  includeStatus?: boolean;
+  /**
+   * returns data points marked with the uncertain status code.
+   * The default behavior of the API is to treat them the same as bad data points and don't returned them.
+   */
+  treatUncertainAsBad?: boolean;
+  /**
+   * denotes that the API should return bad data points.
+   * Because the API treats uncertain data points as bad by default,
+   * this parameter includes both uncertain and bad data points.
+   */
+  ignoreBadDatapoints?: boolean;
+}
+
+export interface DatapointStatus {
+  code: number;
+  symbol: string;
```

Comment:
Symbol can be a combination of [Category](https://developer.cognite.com/dev/concepts/reference/quality_codes/#subcategories)
and [modifier flags.](https://developer.cognite.com/dev/concepts/reference/quality_codes/#modifier-flags) 
For example `BadBrowseNameInvalid, StructureChanged, Low`. 

I initially tried typing it using [template literals](https://www.typescriptlang.org/docs/handbook/2/template-literal-types.html)
but this scenario was too advanced for that, so went with string. 

Code is the number you get by adding all the category and modifier codes together. 
The example above would result in 2153775104 + 32768 + (1024) + 256 = 2153809152. 
[More details here](https://developer.cognite.com/dev/concepts/reference/quality_codes/#status-code-structure)